buildSrc/build.gradle.kts

```diff
@@ -8,16 +8,16 @@ repositories {
 }
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
+    sourceCompatibility = JavaVersion.VERSION_17
+    targetCompatibility = JavaVersion.VERSION_17
 }
 
 dependencies {
-    implementation("com.android.tools.build:gradle:7.2.1")
+    implementation("com.android.tools.build:gradle:8.1.0")
```

Comment:
Latest stable version is [8.1.2](https://mvnrepository.com/artifact/com.android.tools.build/gradle/8.1.2). Can we upgrade to it, please?